minecraft_servers/phoenix/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "server_name": "phoenix",
+  "nice_name": "Phoenix",
+  "direct_ip": "rainbowfurry.com",
+  "server_wildcards": [
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://rainbowfurry.com",
+    "web_shop": "https://rainbowfurry.com"
```

Comment:
remove if you don't have a dedicated website for your shop

Reply:
changed the url to the shop